### DISCUSSION TITLE
What TUI library was used to create `btop`?

### DISCUSSION THREAD

**Discussion by Ashvith10** [aristocratos/btop#638] [General]
Generally, TUI apps made with older libraries, have limited color palette and lacks gradient. I've seen a growing number of TUI libraries in Go and Rust that use full-color, and also support gradients. I wanted to learn about the library that was using in this app to add nice gradients and colors in C++ (and possibly C).

**Comment by deckstose** [comment]
There is no library being used. There is this [issue](https://github.com/aristocratos/btop/issues/472) already
